quic/s2n-quic-core/src/crypto/tls/slow_tls.rs

```diff
@@ -0,0 +1,216 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+use crate::{
+    application,
+    crypto::{tls, CryptoSuite},
+    transport,
+};
+use alloc::{boxed::Box, vec::Vec};
+use core::{any::Any, task::Poll};
+
+pub struct SlowEndpoint<E: tls::Endpoint> {
+    pub server_endpoint: Option<E>,
+    pub client_endpoint: Option<E>,
+}
+
+impl<E: tls::Endpoint> tls::Endpoint for SlowEndpoint<E> {
+    type Session = SlowSession<<E as tls::Endpoint>::Session>;
+
+    fn new_server_session<Params: s2n_codec::EncoderValue>(
+        &mut self,
+        transport_parameters: &Params,
+    ) -> Self::Session {
+        let inner_session = self
+            .server_endpoint
+            .take()
+            .expect("Server should exist")
+            .new_server_session(transport_parameters);
+        SlowSession {
+            defer: 10,
```

Comment:
Why are we picking 10? I think even 1 is probably good enough for most cases. Maybe we should take this as a parameter?

Reply:
How do you feel about 3. I feel like one is too small. Not sure what the purpose of making this configurable would be.